src/applications/appeals/10182/sass/10182-nod.scss

```diff
@@ -121,6 +121,13 @@ dl.review {
   }
 }
 
+/* IE11 hack to fix edit button placement, see
+ * https://github.com/department-of-veterans-affairs/va.gov-team/issues/25108
+ */
+ _:-ms-fullscreen, :root dl.review dd.widget-content.widget-edit .widget-content-wrap {
```

Comment:
Is `calc` safe to use with IE11? https://caniuse.com/calc seems to say it only has partial support

Reply:
I tested in IE11 on my Windows machine and it was working. Looking through the "Known issues" on that caniuse page, the only one that may apply is the generated content one?